src/net/sourceforge/plantuml/klimt/drawing/eps/DriverTextEps.java

```diff
@@ -34,13 +34,13 @@
  */
 package net.sourceforge.plantuml.klimt.drawing.eps;
 
-import java.awt.Color;
-import java.awt.Shape;
+import java.awt.*;
```

Comment:
Here is a minor remark:
- Be careful with global imports `import xxx.*;`

[FYI @arnaudroques]

Reply:
> Here is a minor remark:
> 
>     * Be careful with global imports `import xxx.*;`
> 
> 
> [FYI @arnaudroques]

Thank you for the feedback :)